### PR TITLE
Node configuration

### PR DESCRIPTION
**Pull request by itkg-bfouche**

### CODE REVIEW

File: en/user_guide/node_configuration.rst [hunk 1]
```diff
@@ -0,0 +1,106 @@
+Node configuration
+==================
+
+Here are the options available in the node configuration. Some of them are required
```
Comment (ngilain) [inline en/user_guide/node_configuration.rst]: availables

File: en/user_guide/node_configuration.rst [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+in order to buld a node and must therefore be provided.
```
Comment (ngilain) [inline en/user_guide/node_configuration.rst]: build

File: en/user_guide/node_configuration.rst [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+Url pattern (required)
+----------------------
+
+The url pattern attribute is generated automatically with the name of the page,
+it represents the pattern that will be the URL of the page on the website.
+
+Scheme (required)
+-----------------
+
+The scheme permet de définir par quel protocole la page est accessible
```
Comment (ngilain) [inline en/user_guide/node_configuration.rst]: Maybe to translate :)

File: en/user_guide/node_configuration.rst [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+(http, https, etc...).
+By default Open Orchestra prend le protocole renseigné dans la configuration du site.
```
Comment (ngilain) [inline en/user_guide/node_configuration.rst]: Here too

File: en/user_guide/node_configuration.rst [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+The theme (required)
+--------------------
+
+The theme to use on the page is choosen here.
+
+Show in menus
+-------------
+
+This attribute indicates if the page should appear in a menu.
```
Comment (ngilain) [inline en/user_guide/node_configuration.rst]: in the menus

File: en/user_guide/node_configuration.rst (hunk 1 above)
Comment (juchi) [inline en/user_guide/node_configuration.rst]: An adjective is invariable. But i would actually say "available options" ;)

**Comment by itkg-bfouche** [comment]
@itkg-ngilain @nicolasThal @itkg-nanne

File: en/user_guide/node_configuration.rst [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+Page (required)
+---------------
+
+The page attribute is the name of the node, it is also used to build the url pattern.
+
+Url pattern (required)
+----------------------
+
+The url pattern attribute is generated automatically with the name of the page,
+it represents the pattern that will be the URL of the page on the website.
```
Comment (ngilain) [inline en/user_guide/node_configuration.rst]: As this is a user guide, not a technical guide, maybe an exemple of what is a pattern could help contributors with a non technical profile.

File: en/user_guide/node_configuration.rst [hunk 7]
```diff
@@ -0,0 +1,105 @@
+Node configuration
+==================
+
+Here are the available options in the node configuration. Some of them are required
+in order to build a node and must therefore be provided.
+
+Page (required)
+---------------
+
+The page attribute is the name of the node, it is also used to build the url pattern.
```
Comment (ngilain) [inline en/user_guide/node_configuration.rst:10]: Not sure if the second part has to be mentionned in a user guide, it's more an "underground" technical aspect

Comment (itkg-bfouche) [inline en/user_guide/node_configuration.rst:10]: I think it is important to say that contribute the node name will generate the node url pattern

Comment (juchi) [inline en/user_guide/node_configuration.rst:10]: i agree with Benjamin, as the url pattern can be defined by users

File: en/user_guide/node_configuration.rst (hunk 6 above)
Comment (juchi) [inline en/user_guide/node_configuration.rst]: Just been added in the new commit

**Comment by itkg-bfouche** [comment]
@itkg-ngilain

**Comment by ngilain** [comment]
:+1:

**Comment by itkg-bfouche** [comment]
:+1:

File: en/user_guide/node_configuration.rst [hunk 8]
```diff
@@ -0,0 +1,122 @@
+Node configuration
+==================
+
+Here are the available options in the node configuration. Some of them are required
+in order to build a node and must therefore be provided.
```
Comment (veolys-nanne) [inline en/user_guide/node_configuration.rst:5]: Could you remind what is a node?

File: en/user_guide/node_configuration.rst (hunk 6 above)
Comment (veolys-nanne) [inline en/user_guide/node_configuration.rst]: the name of the page. It represents...

File: en/user_guide/node_configuration.rst (hunk 8 above)
Comment (juchi) [inline en/user_guide/node_configuration.rst:5]: I think it would be a bit heavy in this context. What do you say about adding a link to the node page ?

Comment (veolys-nanne) [inline en/user_guide/node_configuration.rst:5]: ok

**Comment by juchi** [comment]
updated @itkg-nanne

**Comment by veolys-nanne** [comment]
:+1:

File: en/user_guide/node_configuration.rst [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+
+The url pattern attribute is generated automatically with the name of the page.
+It represents the pattern that will be the URL of the page on the website.
+
+If the pattern starts with a slash, it will be appended to the  website's base url
+(domain + language prefix) to make the full page url. Otherwise, if the page has a parent,
+the pattern will be appended to the parent's URL to generate the final URL.
+
+Here is a as example of some URL generated from the page patterns if the website's
+domain is "example.com" and the language is "en".
```
Comment (nicolasThal) [inline en/user_guide/node_configuration.rst:23]: the language is not important, the prefix is used to generate the url

**Pull request merged**